domains/auth/Auth.API/Tests/Controllers/TestTokenController.cs

```diff
@@ -0,0 +1,88 @@
+using System;
+using System.Collections.Generic;
+using API.Configuration;
+using API.Models;
+using API.Services;
+using API.TokenStorage;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Options;
+using Moq;
+using Xunit;
+using Xunit.Categories;
+
+namespace API.Controllers;
+
+[UnitTest]
+public class TestTokenController
+{
+    private readonly Mock<ICookies> cookiesMock = new();
+    private readonly Mock<IOptions<AuthOptions>> authOptionsMock = new();
+    private readonly Mock<ITokenStorage> tokenStorage = new();
+
+    private readonly TokenController tokenController;
+
+    public TestTokenController()
+    {
+        authOptionsMock.Setup(x => x.Value).Returns(new AuthOptions
+        {
+            CookieName = "Authorization",
+        });
+
+        tokenController = new TokenController(
```

Comment:
This looks a little weirdly formatted in terms of line breaks, parantheses, etc.

Reply:
How would you prefer it? I actually think it is quite easy to read when you quickly skim the file

Reply:
```
        tokenController = new (authOptionsMock.Object, tokenStorage.Object)
        {
            ControllerContext = new () { HttpContext = new DefaultHttpContext() }
        };
```

I'd do it something like this. I don't think the extra line breaks help the clarity when the lines can still be this short like this.

Reply:
I think this should be something for the linter to take care of.
Untill we have a linting rule, lets not focus on this.